opencti-platform/opencti-front/src/components/list_lines/ListLinesContent.jsx

```diff
@@ -190,6 +192,8 @@ class ListLinesContent extends Component {
             contentMapping={contentMapping}
             contentMappingData={contentMappingData}
             bypassEditionRestriction={bypassEditionRestriction}
+            isOrganizationAdmin={isOrganizationAdmin}
```

Comment:
where do you need this property ? isOrganizationAdmin, same for organization

Reply:
organization is needed in the line component (SettingsOrganizationUserLine).
isOrganizationAdmin is not used anymore. I will delete it

Reply:
still, it's a bit weird to have `organization` in a generic component used for all entities list. I wonder if there is another way to have this information in SettingsOrganizationUserLine

Reply:
but I see that there are already other spectific properties too...

Reply:
Yep! I agree that it is not the best way

Reply:
Stepping into : you should use the generic entityId attribute of listLines for that